### DISCUSSION TITLE
Backlog of PRs when surpass the prConcurrentLimit

### DISCUSSION THREAD

**Discussion by HaoranLI9** [renovatebot/renovate#22951] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.122.0

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

Hello,

I get 2 questions here.

I've recently been using the Renovate bot on a large repository, and have noticed that the initial "Configure Renovate" pull request, which provides details about upcoming PRs under "What to Expect", is being cut off. I believe there may be a word limit for the description, and given the size of my repository, the bot could potentially generate 1000 PRs. How can I handle this situation? Is there a way to save this detailed information in a txt file through some Renovate configuration? 

Based on my understanding, when I execute renovate/renovate, it scans the entire repository and prepares to execute all necessary updates, but due to the prConcurrentLimit, it can only display a limited number of PRs at once. What happens to the PRs that exceed this limit? Can they be saved or logged elsewhere?

I appreciate your help in resolving these issues.

### What you tried so far.

Hello,

For the limit of description "Configure Renovate", could the RENOVATE_PAGINATE_ALL environment variable be of any help?

For the second question, I find that Renovate is Stateless, so my request can't be satisfied?

I'am looking forward to your  answer.
Thanks

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Debug logs should give you a full listing

**Reply by HaoranLI9** [reply]
Thank you, rarkins, I'am a novice of renovate,  it's very helpful.
